nucypher/policy/collections.py

```diff
@@ -66,7 +67,8 @@ class IsDisorienting(Bob.NotEnoughNodes):
         leaves Bob disoriented.
         """
 
-    node_id_splitter = BytestringSplitter((to_checksum_address, ETH_ADDRESS_BYTE_LENGTH), ID_LENGTH)
+    ursula_con_kfrag_splitter = BytestringSplitter((to_checksum_address, ETH_ADDRESS_BYTE_LENGTH),
```

Comment:
I don't think it's a good idea to insert Spanish into variable names.

Reply:
Yes, I agree, this is temporary, and a pun based on the the original issue concept (#2234).